src/domain/opportunity/opportunity.entity.ts

```diff
@@ -69,6 +73,18 @@ export class Opportunity extends BaseEntity
   )
   projects?: Project[];
 
+  @Field(() => [ActorGroup], {
+    nullable: true,
+    description:
+      'The set of actor groups within the context of this Opportunity',
+  })
+  @OneToMany(
+    () => ActorGroup,
+    actorGroup => actorGroup.opportunity,
+    { eager: true, cascade: true }
```

Comment:
Again, was this eager loading intentional? There will be plenty of opportunities and plenty of actorGroups + actors. This will have similar impact on performance as user-userGroup eager loadings.

Reply:
yes plenty of them, but all narrowly scoped e.g. 2 or 3 per challenge, and then maybe 3 actor groups per opportunity. First operating and can then see where to optimise.